--- sundials-6.6.0/include/cvodes/cvodes.h	2023-09-06 11:30:08
+++ ../patched_sundials/include/cvodes/cvodes.h	2023-09-06 11:29:48
@@ -35,6 +35,11 @@
 /* lmm */
 #define CV_ADAMS          1
 #define CV_BDF            2
+/* SUNDIALS EXTENSION */
+#define CV_DOPRI 3
+#define CV_ExpRK 4
+#define CV_ImpRK 5
+#define CV_CRANI 6
 
 /* itask */
 #define CV_NORMAL         1
@@ -58,7 +63,9 @@
 #define CV_SUCCESS               0
 #define CV_TSTOP_RETURN          1
 #define CV_ROOT_RETURN           2
-
+/* SUNDIALS EXTENSION */
+#define CV_ZERO_DETACH_RETURN    3
+		
 #define CV_WARNING              99
 
 #define CV_TOO_MUCH_WORK        -1
